python/py-ansible-compat/Portfile

```diff
@@ -15,14 +15,19 @@ maintainers         {judaew @judaew} openmaintainer
 description         \
     Functions that help interacting with various versions of Ansible
 long_description    {*}${description}
-homepage            https://github.com/ansible/ansible-compat
 
-checksums           rmd160  ca04c268c763cebfd6b07551376c585b6d484138 \
-                    sha256  b3e9f9d7c3a1ce6222de444e9dc6fece7eba70ac64f2a0befdc4e2d542018b4a \
-                    size    73021
+homepage            https://pypi.python.org/pypi/ansible-compat
+master_sites        pypi:a/ansible_compat
+distname            ansible_compat-${version}
+distfiles           ansible_compat-${version}.tar.gz
+dist_subdir         ansible_compat-${version}
+
+checksums           rmd160  d7f02c83fade368c757f9144dc4bbd5a87a9a5a4 \
+                    sha256  0ad873e0dae8b2de79bc33ced813d6c92c716c4d7b82f9a4693e1fd57f43776e \
+                    size    81818
 
 python.pep517       yes
```

Comment:
this is the default for subports for Python 3.7+ so please remove

Reply:
@reneeotten - to be clear, you're just asking for the removal of the "python.pep517       yes"?